command/login.go

```diff
@@ -156,9 +156,9 @@ func (c *LoginCommand) Run(args []string) int {
 		c.flagField = "token"
 	}
 
-	if c.flagNoStore && c.flagNoPrint {
+	if c.flagNoStore && !c.flagPrintToken {
 		c.UI.Error(wrapAtLength(
-			"-no-store and -no-print cannot be used together"))
+			"-no-store and the absence of -print-token cannot be used together"))
```

Comment:
Should it be "-no-store cannot be used without -print-token" instead?

Also, is this actually the case?